src/CTA.Rules.Actions/ObjectCreationExpressionActions.cs

```diff
@@ -34,5 +36,71 @@ ExpressionSyntax action(SyntaxGenerator syntaxGenerator, ObjectCreationExpressio
             return action;
         }
 
+        public Func<SyntaxGenerator, ObjectCreationExpressionSyntax, ExpressionSyntax> GetReplaceOrAddObjectPropertyIdentifierAction(string oldMember, string newMember, string newValueIfAdding)
+        {
+            Func<SyntaxGenerator, ObjectCreationExpressionSyntax, ExpressionSyntax> action = (SyntaxGenerator syntaxGenerator, ObjectCreationExpressionSyntax node) =>
+            {
+                if(node.Initializer != null && node.Initializer.Expressions.Count > 0)
+                {
+                    var memberList = node.Initializer.Expressions.Where(n => n.Kind() == SyntaxKind.SimpleAssignmentExpression);
+                    if(memberList.Count() > 0)
+                    {
+                        var assignMemberList = memberList.Select(n => (AssignmentExpressionSyntax)n);
+                        var member = assignMemberList.Where(n => n.Left.ToFullString().Contains(oldMember)).FirstOrDefault();
+                        if (member != null)
+                        {
+                            var newExpression = SyntaxFactory.ParseExpression(newMember + "=" + member.Right);
+                            var newNode = node.Initializer.Expressions.Replace(member, newExpression);
+                            node = node.WithInitializer(node.Initializer.WithExpressions(newNode)).NormalizeWhitespace();
+                        }
+                        else
+                        {
+                            var newExpression = SyntaxFactory.ParseExpression(newMember + "=" + newValueIfAdding);
+                            var newNode = node.Initializer.Expressions.Add(newExpression);
+                            node = node.WithInitializer(node.Initializer.WithExpressions(newNode)).NormalizeWhitespace();
+                        }
+                    }
+                }
+                return node;
+            };
+            return action;
+        }
+
+        public Func<SyntaxGenerator, ObjectCreationExpressionSyntax, ExpressionSyntax> GetReplaceObjectPropertyValueAction(string oldMember, string newMember)
+        {
+            Func<SyntaxGenerator, ObjectCreationExpressionSyntax, ExpressionSyntax> action = (SyntaxGenerator syntaxGenerator, ObjectCreationExpressionSyntax node) =>
```

Comment:
This declaration can be simplified. Please run code analysis on the commit to get suggestions from editorconfig.

Reply:
Awesome, thanks Mark. I will always check messages from code analysis from now on. Completed as mentioned.